servers/cromwell/jobs/__main__.py

```diff
@@ -20,7 +20,7 @@
     default=os.environ.get('CROMWELL_URL'))
 parser.add_argument(
     '--use_caas',
-    type=int,
+    type=str,
```

Comment:
Would it work to use type boolean here or else use bool() below like you were doing before?

Reply:
I believe using type boolean here would be the same as doing `bool(value)`. But, if the value is the string `False` it will still get evaluated to true.

Reply:
Ah okay, so better to keep this as str here.